packages/channels/src/telegram/api.ts

```diff
@@ -58,6 +58,22 @@ export class TelegramApi extends ChannelApi<TelegramService> {
   private async handleTelegrafMessage(scope: string, ctx: NarrowedContext<Context<Update>, Update.MessageUpdate>) {
     if ('text' in ctx.message) {
       await this.service.receive(scope, this.extractEndpoint(ctx), { type: 'text', text: ctx.message.text })
+    } else {
+      const type = Object.keys(ctx.message).find((i) => ['photo', 'document', 'video', 'audio'].includes(i))
+
+      if (!type) {
+        return
+      }
+
+      // @ts-ignore
+      // Get best photo
+      const item = type === 'photo' ? ctx.message.photo[ctx.message.photo.length - 1] : ctx.message[type]
```

Comment:
are media file always arrays ? if yes, do I understand correctly that you only send the last item ? is this really the intended behaviour ? 

if yes, then you should do the following `const item = ctx[type].pop()`.

but if you want my opinion, we should send all media elements in either one payload or multiple subsequent payloads

Reply:
I searched for references regarding this, the array contains the same images but in different sizes, so we just care about the last one, since it has the best quality

Reply:
then if photo is the only media with this behaviour, you should have a dedicated if statement to it.

pseudo : 
```ts
if(isText){
//  map to text payload
} else if (isImage) {
//  process photo payload 
} else if (isOtherMedia){
//  process file, video and audio payload 
} else {
// swallow payload because not supported ? 
// throw ? 
// downrender to text ? 
}
```

Reply:
Done